AmethystAPI/src/minecraft/src-client/common/client/gui/gui/VisualTree.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include "UIControl.h"
+
+class VisualTree {
+public:
+    BUILD_ACCESS(this, class UIControl*, root, 0x8);
```

Comment:
This should be replaced with an actual field in the struct, see: https://github.com/FrederoxDev/Amethyst/pull/39#issuecomment-1895228515